### DISCUSSION TITLE
error while integrating dynamic widgets

### DISCUSSION THREAD

**Discussion by BhaskarreddyHS** [algolia/instantsearch#5301] [Q&A]
I am trying to implement DynamicWidgets in my react class component by ending up with this warning . 

"**DynamicWidgets' is not exported from 'react-instantsearch-dom**".

`import React, { Component } from 'react'
import { DynamicWidgets } from 'react-instantsearch-dom'

export default class AlgoliaDynamicWidgets extends Component {
  render() {
    return (
      <DynamicWidgets
        facets={['*']|[]}
        maxValuesPerFacet={20}
      >
        {children}
    </DynamicWidgets>
    )
  }
}
`

**Comment by Haroenv** [comment]
Have you updated your React InstantSearch version? Can you reproduce this error in our sandbox?  We have a template available for that purpose [here](https://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/react-instantsearch). Thanks!
